apis/pipelines/hub/conversions.go

```
@@ -0,0 +1,54 @@
+package v1beta1
+
+import (
+	"encoding/json"
+	"github.com/sky-uk/kfp-operator/apis"
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+)
+
+func AddComponentsToFrameworkParams(tfxComponents string, framework *PipelineFramework) {
+	marshal, _ := json.Marshal(tfxComponents)
```

is this just adding quotes? Also error is ignored

---

And turn into []byte. I did read the implementation of Marshal and on passing in a string it is impossible to return an error so rather than returning I thought just ignore here. Can return if you want

---

will also be escaping special chars, so better to use `json.Marshal` as has been done

---

error handled in https://github.com/sky-uk/kfp-operator/pull/508/commits/ebbf7619d839bbbd45c39b4de20a4bb2c3d1bfdc